src/transport/AdminPairingTable.h

```diff
@@ -167,6 +169,9 @@ class DLL_EXPORT AdminPairingInfo
     uint8_t * mRootCert            = nullptr;
     uint16_t mRootCertLen          = 0;
     uint16_t mRootCertAllocatedLen = 0;
+    uint8_t * mICACert             = nullptr;
+    uint16_t mICACertLen           = 0;
+    uint16_t mICACertAllocatedLen  = 0;
```

Comment:
Why do we care about the allocated lengths? Could we reduce potential state mismatch hazards with just `mICACertLen` and `mOpCertLen` ?